cache-tree.c

```diff
@@ -527,6 +527,8 @@ static void write_one(struct strbuf *buffer, struct cache_tree *it,
 void cache_tree_write(struct strbuf *sb, struct cache_tree *root)
 {
 	write_one(sb, root, "", 0);
+
+	trace2_data_intmax("cache_tree", NULL, "write_size", sb->len);
```

Comment:
Shouldn't this be `uintmax` instead (i.e. `unsigned`)?

Reply:
trace2 only has an _intmax() form.  i'll add casts to these.